_data/api/pro-sante-connect.md

```diff
@@ -0,0 +1,70 @@
+---
+title: API Pro Santé Connect
+tagline: Permettre l'authentification des professionnels de la santé
+external_site: https://esante.gouv.fr/
+is_open: -1 # -1 means API not open
+datapass_link: https://esante.gouv.fr/
+access_page:
+  - who:
+      - Un particulier
+    is_eligible: -1
+    description: |
+      Seules les administrations et les éditeurs de logiciels sont habilitées à utiliser l'API Pro Santé Connect ou intégrer le bouton Pro Santé Connect.
+
+      Si vous cherchez à contacter l'équipe Pro Santé Connect, vous pouvez [contacter l'Agence Numérique en Santé](https://esante.gouv.fr/assistance)
+
+      <Button href="/rechercher-api">Ou vous pouvez rechercher une autre API</Button>
+  - who:
+      - Une entreprise
+    is_eligible: 0
+    description: |
+      Seuls les éditeurs de logiciels proposant une solution web adressée aux professionnels de santé peuvent demander à se raccorder à PRO Santé Connect.
```

Comment:
We miss the condition "french"

Let's choose one spelling for Pro Santé Connect . According to their website I would stick to Pro Santé Connect and avoid PRO